app/src/main/kotlin/com/mapzen/erasermap/presenter/MainPresenter.kt

```diff
@@ -32,6 +32,7 @@ interface MainPresenter {
     var poiTapPoint: FloatArray?
     var poiTapName: String?
     var poiCoordinates: LngLat?
+    var willPauseImmediately: Boolean?
```

Comment:
Maybe a better name can be used here to clarify the context its used, not sure what though!